networkx/algorithms/tests/test_covering.py

```diff
@@ -0,0 +1,89 @@
+from nose.tools import assert_equal, assert_true
+import networkx as nx
+
+r"""Tests for :func:`min_edge_cover`"""
+
+
+def test_min_edge_cover_empty_graph():
+    G = nx.Graph()
+    assert_equal(nx.min_edge_cover(G), set())
+
+
+def test_min_edge_cover_graph_with_loop():
+    G = nx.Graph()
+    G.add_edge(0, 0)
+    assert_equal(nx.min_edge_cover(G), set([(0, 0)]))
```

Comment:
It's easier to read set literals, and they match the mathematical notation better anyway: `{(0, 0)}`.